docs/swagger_v3/stats.spec.yaml

```diff
@@ -426,3 +459,21 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+  /stats:
+    get:
+      deprecated: true
+      description: Deprecated. Use /totalstats or /deltastats instead.
```

Comment:
This is not deprecated, the /stats endpoint provides data that /totalstats or /deltatstats doesn't provide

Reply:
My bad, mistakenly read the comment above the v1 version.